### ISSUE TITLE
functions:shell not working

### ISSUE THREAD

**Issue by snj33v** [firebase/firebase-tools#1243]
### [REQUIRED] Environment info

firebase-tools: 6.9.0

Platform: Linux

### [REQUIRED] Test case

works on default project from `firebase init functions` 

### [REQUIRED] Steps to reproduce

same as mentioned in docs, for explantion i'm using `firebase functions:shell`
### [REQUIRED] Expected behavior

```
i  functions: Preparing to emulate functions.
✔  functions: helloWorld
firebase > helloWorld
{ [Function]
  get: [Function],
  head: [Function],
  post: [Function],
  put: [Function],
  patch: [Function],
  del: [Function],
  delete: [Function],
  cookie: [Function],
  jar: [Function],
  defaults: [Function] }
```

### [REQUIRED] Actual behavior
```
✔  functions: Emulator running at http://localhost:5001
firebase > helloWorld
ReferenceError: helloWorld is not defined
```

### Note
this works on 6.8.0 though
new version(6.9.0) does not display url too `http://localhost:5000/fresh-con/us-central1/helloWorld`
even default port is changed to 5001

**Comment by snj33v** [comment]
#1214

**Comment by samtstern** [comment]
@snj33v thanks it looks like we broke something with the new Functions emulator we introduced in `6.9.0`.  We'll investigate ASAP.

**Comment by samtstern** [comment]
@abehaskins `functions:shell` has this code block:

```
      _.forEach(emulator.triggers, function(trigger) {
        if (_.includes(emulator.emulatedFunctions, trigger.name)) {
          var localFunction = new LocalFunction(trigger, emulator.urls, emulator.controller);
          var triggerNameDotNotation = trigger.name.replace(/\-/g, ".");
          _.set(replServer.context, triggerNameDotNotation, localFunction.call);
        }
      });
```

But I don't think the `FunctionsEmulator` object exposes any of those trigger fields it's looking for.

**Comment by abehaskins** [comment]
Investigating now, in the meantime feel free to continue your work with `npm install firebase-tools@6.8.0`

**Comment by geminiyellow** [comment]
woo, i upgrade firebase-tools and everything is down. got. please fix it.

**Comment by Cdik** [comment]
Same here. 
I don't know if it's linked but I also noticed that installing the latest `firebase-tools` throw an error regarding `fsevents@1.2.9` which isn't thrown with `firebase-tools@6.8.0`

**Comment by samtstern** [comment]
I am working on a fix in #1260.

**Comment by Cdik** [comment]
I guess this has been fixed with [6.9.1](https://github.com/firebase/firebase-tools/releases/tag/v6.9.1)

**Comment by samtstern** [comment]
Sorry but the fix for this issue wasn't ready for 6.9.1, hoping for the
next release.

*From: *Cdik <notifications@github.com>
*Date: *Thu, May 9, 2019, 8:48 AM
*To: *firebase/firebase-tools
*Cc: *Sam Stern, Assign

I guess this has been fixed with 6.9.1

**Comment by illuminist** [comment]
As of 6.9.2 this issue has been fixed.

**Comment by samtstern** [comment]
This has been fixed in #1260 and will be part of the next release.
